### DISCUSSION TITLE
每次启动都会提示“您要应用程序“YesPlayMusic.app”接受传入网络连接吗”

### DISCUSSION THREAD

**Discussion by BrahmingWu** [qier222/YesPlayMusic#1587] [Q&A]
应用版本：0.4.4-1
OS版本：macOS 12.3.1

每次启动都会提示这个：
<img width="246" alt="CleanShot 2022-04-11 at 16 10 41@2x" src="https://user-images.githubusercontent.com/10195904/162693235-e87df6c9-2a2b-4573-bd0a-c10901d4dcbb.png">

设置中已经允许：
![CleanShot 2022-04-11 at 16 12 26@2x](https://user-images.githubusercontent.com/10195904/162693598-65b4291a-2849-4887-8a88-acf70c84b56b.png)

首页控制台输出：
<img width="1325" alt="CleanShot 2022-04-11 at 16 14 02@2x" src="https://user-images.githubusercontent.com/10195904/162693779-87d80e4c-8cd8-47f4-968c-bb8530f080c1.png">

**Comment by pan93412** [comment]
感覺是設定問題⋯⋯我用 LuLu Firewall 沒有類似問題。

**Comment by BrahmingWu** [comment]
> 感覺是設定問題⋯⋯我用 LuLu Firewall 沒有類似問題。

應該是吧，是之前用著就沒問題，這次是我重新安裝了macOS，新系統安裝的這個。好奇怪

**Comment by pan93412** [comment]
試試看重新添加 `YesPlayMusic.app`？

**Comment by DelayNooMore** [comment]
请问你现在还有这个问题吗,我也遇到了,刚刚升级了一下软件,现在每次打开都提示这个弹窗.
系统:macos 12.3
v0.4.4和v0.4.4-1都会提示这个.

**Comment by Lematouu** [comment]
同样的问题，是不是证书有问题？？？

**Comment by qier222** [comment]
> 同样的问题，是不是证书有问题？？？

没有证书
